platforms/hyperledger-besu/configuration/add-validator.yaml

```diff
@@ -71,6 +71,19 @@
     loop_control:
       loop_var: organizationItem
     when: add_new_org == 'true'
+
+  #installs cert-manager
```

Comment:
This can be renamed to `# Install cert-manager` (space between # and installs and I in caps)

Reply:
fixed in pr #2187 as unwanted files got added while squashing